src/pages/documents/[id].tsx

```diff
@@ -52,11 +63,11 @@ const scrollToPassage = (index: number) => {
 const DocumentPage: InferGetServerSidePropsType<typeof getServerSideProps> = ({ document, family }: TProps) => {
   const [passageIndex, setPassageIndex] = useState(null);
   const [passageMatches, setPassageMatches] = useState<TPassage[]>([]);
+  const [totalNoOfMatches, setTotalNoOfMatches] = useState(0);
   const router = useRouter();
   const startingPassage = Number(router.query.passage) || 0;
-  const { status, families, searchQuery } = useSearch(router.query, null, document.import_id, !!router.query[QUERY_PARAMS.query_string]);
+  const { status, families, searchQuery } = useSearch(router.query, null, document.import_id, !!router.query[QUERY_PARAMS.query_string], 500);
```

Comment:
The `500` ... the param is `NoOfPassagesPerDoc` ... but the `max_passages_per_doc` is defined as 50 earlier on?

Reply:
This is so that we can optionally override the default in some given situations. Within the context of a document, we want as many passages returned as possible. I will replace this with the constant to make it clearer!